### DISCUSSION TITLE
I need to setup Crossplane DR in multi-region.

### DISCUSSION THREAD

**Discussion by Harshith-Krishnappa96** [crossplane/crossplane#4519] [Q&A]
Hi Team,

Help me with the best approach as I am new to Crossplane, Can we setup DR for Crossplane?
If yes, please help me with best approach as I need to fully automate the DR process for Crossplane.

**Comment by jeanduplessis** [comment]
Are you aware of this post? https://aws.amazon.com/blogs/opensource/disaster-recovery-when-using-crossplane-for-infrastructure-provisioning-on-aws/

**Comment by Harshith-Krishnappa96** [comment]
Ya I went through this document. Is there an alternate way or a simpler/better way to achieve this. Thanks @jeanduplessis for your response on this
